src/utils/helpers.test.js

```diff
@@ -0,0 +1,58 @@
+import { getUKFormattedEuros } from './helpers';
+
+describe('getUKFormattedEuros', () => {
+  it('should throw an error if no argument is passed', () => {
+    function passZeroArgs() {
+      getUKFormattedEuros();
+    }
+    expect(passZeroArgs).toThrowError('undefined is not an integer');
+  });
+  it('should throw an error if passed a value that is not an integer', () => {
+    function passString() {
+      getUKFormattedEuros('any string');
+    }
+    function passFloat() {
+      getUKFormattedEuros(1.1);
+    }
+    expect(passString).toThrowError('any string is not an integer');
+    expect(passFloat).toThrowError('1.1 is not an integer');
```

Comment:
i like that you are experimenting with different styles for better readability. in this case: what do you think reads better?

```js
function passString() {
  getUKFormattedEuros('any string');
}
function passFloat() {
  getUKFormattedEuros(1.1);
}
expect(passString).toThrowError('any string is not an integer');
expect(passFloat).toThrowError('1.1 is not an integer');

// versus

expect(getUKFormattedEuros('any string')).toThrowError('any string is not an integer');
expect(getUKFormattedEuros(1.1)).toThrowError('1.1 is not an integer');
```

Reply:
[Jest requires wrapping the function call in another function](https://jestjs.io/docs/expect#tothrowerror) in order for the error to be caught, so the alternative would be:

```JavaScript
expect(() => {
  getUKFormattedEuros('any string');
}).toThrowError('any string is not an integer');
expect(() => {
  getUKFormattedEuros('1.1');
}).toThrowError('1.1 is not an integer');
```
I personally find the the first one easier to read

Reply:
ah got ya. sorry i didnt know this about jest. i think the arrow functions are better, but this is really just minor. your call.